src/code/code_0x8012EC80.c

```diff
@@ -229,64 +229,42 @@ s32 func_8012EE34(s16 arg0) {
     return 0;
 }
 
-/* static */ u16 D_801C2380[12][6] = {
+static u16 D_801C2380[6][12] = {
     {
         SCENE_00KEIKOKU,
         SCENE_BOTI,
         SCENE_13HUBUKINOMITI,
         SCENE_24KEMONOMITI,
         SCENE_IKANAMAE,
         SCENE_TOWN,
-    },
-    {
         SCENE_ICHIBA,
         SCENE_BACKTOWN,
         SCENE_CLOCKTOWER,
         SCENE_ALLEY,
         -1,
     },
-    {
-        SCENE_20SICHITAI,
-        SCENE_20SICHITAI2,
-        SCENE_21MITURINMAE,
-        SCENE_22DEKUCITY,
-        SCENE_DEKU_KING,
-        SCENE_KINSTA1,
-    },
-    { -1 },
+    { SCENE_20SICHITAI, SCENE_20SICHITAI2, SCENE_21MITURINMAE, SCENE_22DEKUCITY, SCENE_DEKU_KING, SCENE_KINSTA1, -1 },
```

Comment:
Formatter did something weird here. Maybe try removing some trailing commas and re-run?

Reply:
I can try again, although first try didn't seem to help.